docs/dev/plugins/cloudFoundry/deploy-cf-bosh-lite.md

```diff
@@ -0,0 +1,161 @@
+# Deploying CloudFoundry using BOSH-Lite
+
+## Setup
+
+This guide assumes the following Prerequisites:
+- VT-x or AMD-v virtualization enabled
+- At least 16 GB of RAM
+- Ubuntu 16.04 as the Host OS
+
+### Deploying BOSH-Lite
+
+#### Installing VirtualBox
+
+```bash
+echo "deb http://download.virtualbox.org/virtualbox/debian xenial contrib" | sudo tee -a /etc/apt/sources.list
+wget -q https://www.virtualbox.org/download/oracle_vbox_2016.asc -O- | sudo apt-key add -
+wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | sudo apt-key add -
+sudo apt-get update
```

Comment:
maybe there is also an upgrade necessary or useful

Reply:
Nope. Performing a system upgrade aka `sudo apt-get upgrade -y` is not really necessary. 

The `apt-get update` is required to reload the package lists. Thats needed to find the latest version of VirtualBox